classes/ProductAssembler.php

```diff
@@ -55,18 +55,70 @@ public function __construct(Context $context)
      */
     private function addMissingProductFields(array $rawProduct): array
     {
+        $sql = $this->getSqlQueryProductFields([(int) $rawProduct['id_product']]);
+        $rows = Db::getInstance()->executeS($sql);
+        if ($rows === false) {
```

Comment:
I believe `executeS` returns false only if there is an error
```suggestion
        if (!is_array($rows) || empty($rows)) {
```

Reply:
I used `empty($rows)` only, the function will return array or false, if the value is empty, it's a non empty array.